Source/WebKit/webpushd/PushService.mm

```diff
@@ -521,6 +523,23 @@ void reject(WebCore::ExceptionData&& data)
     });
 }
 
+void PushService::setPushesEnabledForBundleIdentifierAndOrigin(const String& bundleIdentifier, const String& securityOrigin, bool enabled, CompletionHandler<void()>&& handler)
+{
+    if (bundleIdentifier.isEmpty() || securityOrigin.isEmpty()) {
+        RELEASE_LOG_ERROR(Push, "Ignoring setPushesEnabledForBundleIdentifierAndOrigin request with bundleIdentifier (empty = %d) and securityOrigin (empty = %d)", bundleIdentifier.isEmpty(), securityOrigin.isEmpty());
+        return handler();
+    }
+
+    m_database->setPushesEnabledForOrigin(bundleIdentifier, securityOrigin, enabled, [this, handler = WTFMove(handler)](bool recordsChanged) mutable {
+        if (!recordsChanged)
+            return handler();
+
+        updateTopicLists(m_connection, m_database, [handler = WTFMove(handler)]() mutable {
```

Comment:
Wouldn't this work?
`updateTopicLists(m_connection, m_database, WTFMove(handler));`

Reply:
Yep, will fix this.